### PR TITLE
Add Alias to allow one to easily map one value to another

### PR DESCRIPTION
**Pull request by devotox**
- For times when we have deprecated values i.e. has_single_point_of_contact -> use_single_point_of_contact but we still want to use the old key for things like the ios app

### CODE REVIEW

File: lib/model.js [hunk 1]
```diff
@@ -161,6 +163,49 @@ module.exports = class Model {
 		return this;
 	}
 
+		/**
```
Comment (rafaelalvessa) [inline lib/model.js:166]: Fix indentation.

File: lib/model.js [hunk 2]
```diff
@@ -161,6 +163,49 @@ module.exports = class Model {
...
+	 * @return { object } [ returns the options definiing the model ]
```
Comment (rafaelalvessa) [inline lib/model.js:167]: Fix "definiing" typo.

File: lib/model.js [hunk 3]
```diff
@@ -187,20 +232,35 @@ module.exports = class Model {
...
 
+		let isInput = this.getOption('input');
+		let isOutput = this.getOption('output');
+
+		let reduceAlias = (a, b) => [ a,  this.getValue(b) ].join(' ').trim();
+
 		set.forEach((key) => {
 			let attribute = this.getAttribute(key);
 
+			if( attribute && (
```
Comment (rafaelalvessa) [inline lib/model.js:243]: Remove space before `attribute`.

File: lib/model.js [hunk 4]
```diff
@@ -251,11 +311,21 @@ module.exports = class Model {
...
 
+		let isInput = this.getOption('input');
+		let isOutput = this.getOption('output');
+
 		Object.keys(attributes).forEach((key) => {
-			let attr = this.getAttribute(key);
-			let error = this.test(attr, key);
+			let attribute = this.getAttribute(key);
+
+			if( attribute && (
```
Comment (rafaelalvessa) [inline lib/model.js:323]: Remove space before `attribute`.

File: lib/model.js (hunk 1 above)
Comment (thehogfather) [inline lib/model.js:166]: strange indent

**Review by thehogfather — APPROVED** [review]

**Pull request merged**